modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/setDataRecord.es.js

```diff
@@ -48,7 +48,8 @@ export default function setDataRecord(
 
 		if (
 			!edited &&
-			Liferay.ThemeDisplay.getDefaultLanguageId() === languageId
+			Liferay.ThemeDisplay.getDefaultLanguageId() === languageId &&
+			(value === '' || !value.length || value === false)
```

Comment:
Hi @alinedoleron! Just a doubt I have, is it possible for this value to be false (a boolean)? What is exactly this value?

Reply:
Hi @rebsilva! Yes, for the Boolean field. I need to remove the translations values that wrongly comes by default for the site's language. So besides checking the field was not edited I needed to check if the value is not empty string, empty array or false (for the boolean field). But if you have a suggestion to improve this fix it will e very welcome. 🌷 
![boolean](https://github.com/liferay-forms/liferay-portal/assets/26491796/ec397e35-cc76-4fd8-88e6-be46bbcfc143)

Thanks!

Reply:
I see @alinedoleron! Thanks for you explanation, the fix is fine to me! :blush: